.changeset/healthy-comics-drive.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog': patch
+---
+
+Display the owner and system as links to the entity pages in the about card.
```

Comment:
Not domain (when applicable)?

Reply:
Not sure. Domains are only available on system entities. Right now the about card feels only usable for component entities, every other entity kind is not supported that well. But it's still used for other entity kinds. Two options:
* Either we have a custom about card per entity kind in the future (this one feels more extensible, even for custom ones)
* Or we support different kinds in the about card and hide the "domain" entry for non systems. 

What do you think?

Reply:
Not an UI expert, but previously had some vague thoughts about generalizing the concept of "things that you are a part of".

```
Part Of
 └ [componenticon] Component <component name that is a monorepo component>
 └ [systemicon] System <system name>
 └ [domainicon] Domain <doman name>
```

or something like that, prominently and with clear visual language

Reply:
I would prefer not to change the general UI right now to keep in line with the current concept (even though I think your feedback is correct, generalized would be better).

I gave the PR an update. Can you take a look?

I now display Domain for system entities only and hide some of the other fields that don't make sense on the kind. Like system on domains, or lifecycle on domains, system and resources.